puncover/cppsupport.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+# See https://blog.flameeyes.eu/2010/06/c-name-demangling/ for context
+#
+# This solution courtesy of:
+# https://stackoverflow.com/questions/6526500/c-name-mangling-library-for-python/6526814
+# (slightly modified to return a dict instead of a list)
+def unmangle(names):
+    args = ['c++filt']
```

Comment:
for the other binaries such as `objdump` the tool respects the toolchain dir that passed, e.g. https://github.com/HBehrens/puncover/blob/7896368183b93afd7db512c8cb29ddb0bab6fce6/puncover/collector.py#L261 Also, shouldn't this be `arm-none-eabi-c++filt`?

In the future, I would like to add another argument that helps distinguishing between `xyz` and `arm-none-eabi-xyz`. Centralizing the command line tooling already would help me doing so.